### ISSUE TITLE
Config.js

### ISSUE THREAD

**Issue by TonybynMp4** [xFutte/futte-newspaper#14]
i tried modifying the config.js but nothing changes in game, no error.

Config.js :
`export const Config = {
	newspaperTitle: 'Weazel-News',
	tabs: {
		showPrisonSentences: false,
		showCityNews: true,
	},
	articles: {
		showImage: true,
		showTitle: true,
		showDate: true,
		showPublisher: true,
		titleMaxLength: 60,
	},

	publishArticleControls: [
		['bold', 'italic', 'underline', 'strike'],
		['blockquote', 'image'],
		[{ list: 'ordered' }, { list: 'bullet' }],
	],
	// Remember to align this with QBShared.Jobs
	reporter: [
		{
			grade: 0,
			canPublish: true,
			canEdit: true,
			canDelete: false,
		},
		{
			grade: 1,
			canPublish: true,
			canEdit: true,
			canDelete: false,
		},
		{
			grade: 2,
			canPublish: true,
			canEdit: true,
			canDelete: true,
		},
	],
	prisonSentences: {
		imageUrl:
			'https://cdn.realsport101.com/images/ncavvykf/gfinityesports/94d9c2c9e240b6b4e792a705ead0a0d188c1af47-808x455.png?w=686&h=386&auto=format',
		imageCaption: 'Bolingbroke Penitentiary',
	},
	text: {
		tabs: {
			newspaper: 'Journal',
			prisonSentences: 'Prison sentences',
			reporterActions: 'Actions Journaliste',
		},
		prisonSentences: {
			title: 'Prison sentences',
			noSentencesAvailable: 'No sentences available',
		},
		reporterActions: {
			title: 'Actions Journaliste',
			noPermissions: 'Vous n\'avez pas les permissions.',
			publishNewStory: 'Publier un nouvel article',
			updateStories: 'Mettre à jour des articles',
			deleteStories: 'Supprimer des articles',
			publishStory: {
				textareaPlaceholder: 'Contenu de l\'article',
				imagePlaceholder: 'Image de l\'article (optionnel)',
				titlePlaceholder: 'Titre de l\'article',
				publish: 'Publier',
				update: 'Mettre à jour',
				discardChanges: 'Annuler',
				preview: 'Prévisualiser',
				wrongImageFormat:
					'L\'image doit être au format .jpg, .jpeg, .png. .webp, .avif, .gif, ou .svg',
				required: 'Ce champ est obligatoire',
				cancel: 'Annuler',
			},
		},
		articles: {
			writtenBy: 'Écrit par',
			on: 'le',
			latestStories: 'Derniers articles',
		},
	},
};
`
In Game:
![image](https://user-images.githubusercontent.com/97451137/182034305-537c15c1-5089-4931-8a0a-47e9927d3c94.png)

**Comment by xFutte** [comment]
Are you compiling when changing the code, as stated in the README.md file? Otherwise, no changes will go through.

**Comment by TonybynMp4** [comment]
Oh i didn't think changing the config required that lol my bad i'll try again x)

**Comment by xFutte** [comment]
Let me know the outcome. FYI, everything gets compiled to the bundle.js file in the /public folder when compiled.

**Comment by TonybynMp4** [comment]
looks like it works !

**Comment by TonybynMp4** [comment]
huh looks like disabling the prison sentence is making the reporter's tab the prison tab 
![image](https://user-images.githubusercontent.com/97451137/182046781-3e52307a-0990-46f4-adbc-a0937b7382ac.png)

**Comment by xFutte** [comment]
@TonybynMp4 Are you sure you are not just on duty as a reporter?

**Comment by TonybynMp4** [comment]
> @TonybynMp4 Are you sure you are not just on duty as a reporter?

I am.. if i wasn't i couldn't see the reporter's tab 🤔

**Comment by xFutte** [comment]
> > @TonybynMp4 Are you sure you are not just on duty as a reporter?
> 
> I am.. if i wasn't i couldn't see the reporter's tab 🤔

Yeah, I think I can see the issue in the code. I'll see if I can find time to fix it today.
